### PR TITLE
feat(docs): adds greenhouse architecture description

### PR DESCRIPTION
**Pull request by ArtieReus**
# Summary

Greenhouse architecture description

# Changes Made

- Proposal

# Related Issues

- Issue 1: [link to issue]

# Screenshots (if applicable)

# Testing Instructions

1. `npm i`
2. `npm run TASK`

# Checklist

- [ ] I have performed a self-review of my code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have made corresponding changes to the documentation (if applicable).
- [ ] My changes generate no new warnings or errors.

### CODE REVIEW

**Review by barsukov — COMMENTED** [review]
Do we need to put this inside of the project that we have already it in EPIC?

File: docs/proposals/003_greenhouse.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+## 7. Next Steps
+
+The next step is to delve deeper and provide a more detailed description of each part of the dashboard.
+
+---
+
+## 8. References
+
+- [Develop Modular Architecture for Aurora Dashboardw](https://github.com/orgs/cloudoperators/projects/4/views/9?sliceBy%5Bvalue%5D=Juno+Core&pane=issue&itemId=80084048)
```
Comment (barsukov) [inline docs/proposals/003_greenhouse.md:129]: I think we can skip reference now cause Aurora is not the name anymore...

**Review by barsukov — APPROVED** [review]
I mean for docs purpose we could keep it here. But this thing need to be proper reworked along with routing concept under carbon app.

**Review by andypf — COMMENTED** [review]
"This document will:

Explain the Greenhouse Dashboard's architecture in detail, including the Core System, Core Modules, and Extensions."

Does it?

File: docs/proposals/003_greenhouse.md (hunk 1 above)
Comment (ArtieReus) [reply docs/proposals/003_greenhouse.md:129]: ok

**Comment by ArtieReus** [comment]
The targe of this document is to start documenting how it should be. Of course this need to be extended but this is the start. You all are welcome to contribute.

**Review by andypf — APPROVED** [review]
👍

**Pull request merged**